docs/user-guide/annotations.md

```diff
@@ -1,66 +1,65 @@
-# Adding Annotations 
+# Annotations 
 ********************
 
 [Mapstore](https://mapstore.geosolutionsgroup.com/mapstore/#/) lets you enrich the map with special features which expose additional information, mark particular position on the map and so on.
 Those features make up the so called **Annotations** layers.
 
-Starting from a new map or an already existing one:
+Starting from a new map or an already existing one, the editor can access the **Annotations** <img src="../img/button/annotations2.jpg" class="ms-docbutton" style="max-height:30px;"/> button from the [TOC](toc.md) <img src="../img/button/show-layers.jpg" class="ms-docbutton" style="max-height:30px;"/> panel on the top-left corner of the map viewer. The annotation panel will open:
```

Comment:
please add here an image of the toolbar
like
![image](https://user-images.githubusercontent.com/11991428/191512574-ade22142-df22-4c0e-8736-e92dc3c1d6e6.png)